wbooks/src/components/book-card/image/Image.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { styles } from './Image.styles';
+
+class Image extends Component {
+  render() {
+    if (this.props.image_url) {
+      return (
+        <img
+          style={styles.image}
+          alt={'book_' + this.props.bookId}
+          src={this.props.image_url}>
+        </img>
+      );
+    } else {
+      return (
+        <div style={styles.image}>
+          <img
+            style={styles.defaultImage}
+            alt={'book_' + this.props.bookId}
+            src={require('../../../images/default_book.svg')}>
+          </img>
+        </div>
+      );
+    }
+  }
+}
+
+Image.propTypes = {
+  bookId: PropTypes.number,
```

Comment:
book id should be a required
`bookId: PropTypes.number.isRequired`
So this call your atention if component receive a null, diferent or undefined param.